src/Compilers/CSharp/Portable/Symbols/Synthesized/GeneratedNames.cs

```diff
@@ -496,6 +496,12 @@ internal static string LambdaCopyParameterName(int ordinal)
             return "<p" + StringExtensions.GetNumeral(ordinal) + ">";
         }
 
+        internal static string AnonymousDelegateParameterName(int index)
+        {
+            // Consistent with `System.Action` and `System.Func` types.
+            return "arg" + StringExtensions.GetNumeral(index + 1);
```

Comment:
We'll want `arg` instead of `arg1` for single parameter case.

Reply:
Thanks, I should have checked the spec first. Updated: 51326d4c86f4e2db1d69ac903466840687976189